libraries/apollo-ast/src/test/kotlin/com/apollographql/apollo3/graphql/ast/test/AntlrParseTest.kt

```diff
@@ -18,11 +18,11 @@ class AntlrParseTest {
       ab bc
       """.trimIndent()
           .buffer()
-          .parseAsGQLDocument()
+          .parseAsGQLDocument(useAntlr = false)
```

Comment:
Name of the class should be changed I guess

Reply:
Yep, I have a follow up PR with more testing, trying to catch some lexer errors. I'll rename in that PR.